scene/main/viewport.cpp

```diff
@@ -3080,7 +3080,7 @@ void Viewport::unhandled_input(const Ref<InputEvent> &p_event, bool p_local_coor
 	get_tree()->_call_input_pause(unhandled_input_group, "_unhandled_input", ev, this);
 
 	// Unhandled key Input - used for performance reasons - This is called a lot less then _unhandled_input since it ignores MouseMotion, etc
```

Comment:
```suggestion
	// Unhandled key Input - used for performance reasons - This is called a lot less than _unhandled_input since it ignores MouseMotion, etc.
```
Not really related, but since you are modifying this code anyways...